npm/pkg/controlplane/controllers/v2/networkPolicyController.go

```diff
@@ -255,7 +258,17 @@ func (c *NetworkPolicyController) syncAddAndUpdateNetPol(netPolObj *networkingv1
 	}
 
 	// install translated rules into kernel
-	npmNetPolObj := translation.TranslatePolicy(netPolObj)
+	npmNetPolObj, err := translation.TranslatePolicy(netPolObj)
+	if err != nil {
+		if errors.Is(err, translation.ErrUnsupportedNamedPort) || errors.Is(err, translation.ErrUnsupportedNegativeMatch) {
+			// We can safely suppress unsupported network policy because re-Queuing will result in same error
+			klog.Warningf("NetworkPolicy %s in namespace %s is not translated because it has unsupported translated features of Windows.", netPolObj.ObjectMeta.Name, netPolObj.ObjectMeta.Namespace)
+			return nil
+		}
+		klog.Errorf("Failed to translate podSelector in NetworkPolicy %s in namespace %s: %s", netPolObj.ObjectMeta.Name, netPolObj.ObjectMeta.Namespace, err.Error())
+		return errNetPolTranslationFailure
```

Comment:
no preference: should we wrap the translatiion error for better context?
`npmerrors.SimpleErrorWrapper("failed to translate network policy", err)`

Reply:
i mean we can see this error message in logs anyway. I did not want to spam the same message multiple times.